### DISCUSSION TITLE
Dịch "matrix factorization" và "factorization machine"

### DISCUSSION THREAD

**Discussion by trgiangdo** [d2l-ai/d2l-vi#2684] [Terminology]
Từ này theo nhiều văn bản sử dụng tương đương với từ `matrix decomposition`.

Một số văn bản Tiếng Việt em thấy có dịch thành `phân rã ma trận`, một số lại dịch là `phân tích ma trận thành nhân tử`.
Em nghĩ nên thống nhất cách dịch từ này trong chương Rec-Sys.

cc: @minhduc0711 @cuongvng @lkhphuc @davidnvq @rootonchair

**Comment by quangvnai** [comment]
Anh thấy **phân rã ma trận** hợp lý hơn. Có một vài tài liệu chính thống xuất hiện từ này, điển hình như [bài báo này](https://sj.ctu.edu.vn/ql/docgia/download/baibao-50436/13-LE%20NGOC%20QUYEN(96-102)013.pdf). 

Từ **phân tích ma trận thành nhân tử** giống như cách giải thích chi tiết hơn. Và cũng có từ như **single value decomposition**, hay **LU decomposition/factorization** cũng xài chung phần đầu như vậy nên đôi khi hơi nhầm lẫn.

**Reply by minhduc0711** [reply]
hmm cá nhân mình thích option còn lại hơn, nhưng nếu dịch SVD và LU decomposition theo như vậy thì đúng là cồng kềnh thật

**Reply by trgiangdo** [reply]
Thực ra thì cụm `SVD` với `LU decomposition` hình như không xuất hiện trong cuốn này thì phải. Tuy nhiên đúng là đa số các bài báo tiếng Việt đều sử dụng cụm `phân rã ma trận`.

**Comment by cuongvng** [comment]
`phân tách` thì sao? `phân rã` nghe hơi nguyên tử :v

**Reply by minhduc0711** [reply]
yeah như vậy cũng hay, mỗi tội hình như không có ai dùng

**Comment by trgiangdo** [comment]
Hiện em đang dịch `matrix decomposition` là `phân rã ma trận`.
Mọi người nghĩ sao nếu dịch `factorization machine` thành `máy phân rã ma trận` hoặc `máy phân rã`?
Em thấy `máy phân tích nhân tử` nghe ổn hơn.

**Reply by cuongvng** [reply]
nghe hơi lạ, nhưng chắc nghe nhiều là quen thôi.

**Comment by lkhphuc** [comment]
* `Decomposition` mình cứ dịch sát là `phân rã` hoặc `phân tách`.

* `Factorization` đã có nghĩa gốc chính là `tìm thừa số`, `phân tích thành nhân tử` hoặc `thừa số hoá`, `nhân tử hoá` rồi.
`Factorization machine` là `máy tìm thừa số` được không? (Từ này lạ quá mình chưa nghe bao giờ :))

**Reply by trgiangdo** [reply]
Em nghĩ nên dịch thống nhất `Matrix factorization` và `Factorization machine`, tức là cùng là `phân rã ma trận - máy phân rã ma trận`, hoặc `phân tích ma trận thành nhân tử - máy phân tích nhân tử`, kiểu kiểu vậy. Còn `Matrix decomposition` không xuất hiện trong cuốn này, chỉ là `Matrix factorization` trong các tài liệu khác thường được coi là đồng nghĩa với `Matrix decomposition`.

Tuy nhiên em chưa tìm được 1 tài liệu Tiếng Việt nào đã dịch từ `Factorization machine` ra nên không biết dịch sao cho ổn.

**Comment by quangvnai** [comment]
@trGiang99 e copy cả câu chứa cụm trên ra đây với em.

**Reply by trgiangdo** [reply]
Cả hai đều là tiêu đề anh ạ.
Còn có 1 câu thế này.
>The strengths of factorization machines over the linear regression and matrix factorization are:

Hiện em đang dịch là:

> Điểm mạnh của máy phân rã ma trận so với hồi quy tuyến tính và phân ra mã trận là

**Reply by quangvnai** [reply]
trong ngữ cảnh này anh nghĩ nên dịch nốt thành `máy phân rã ma trận` cho đồng điều với các từ đi cùng ;)
